graphql-cli/src/main/java/io/ballerina/graphql/cmd/GraphqlCmd.java

```diff
@@ -174,20 +178,27 @@ public void execute() {
                 exit(EXIT_CODE_0);
                 return;
             }
-            if (inputPath == null || inputPath.isEmpty()) {
+            
+            if (inputPath == null || inputPath.trim().isEmpty()) {
```

Comment:
No need to trim here as well. Keep the existing code as it is. (`isEmpty` should suffice in this case as the PicoCLI handles blank strings.